Sources/BrowserServicesKit/Statistics/VariantManager.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  VariantManager.swift
+//  DuckDuckGo
+//
+//  Copyright © 2022 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public enum FeatureName: String {
+
+    // Used for unit tests
+    case dummy
+}
+
+public protocol VariantManager {
```

Comment:
There's a few things like this and the statistics store protocol that are moved from iOS, right? 
I'm good with that since I think in all the cases it's stuff that macOS could benefit from at some point the future, but did want to suggest an alternative for the future: typically if there's info something in BSK needs from the client we usually have some kind of delegate back to the client that returns the info. Like everything else it's not really an official policy or decision, but I think it's become a thing implicitly

Re StatisticsStore it looks like we now have a different version of in all three repos, so in that case I'd either change it to use the above pattern, or better yet and if it makes sense to share it, change iOS and macOS to both use the one now in BSK

Reply:
>  typically if there's info something in BSK needs from the client we usually have some kind of delegate back to the client that returns the info.

I said it elsewhere, but actually I disagree with this as we just end up creating a spaghetti architecture. Obviously not the place to debate it so let's get it on the team agenda soon.